src/mrinufft/io/siemens.py

```diff
@@ -42,47 +44,55 @@ def read_siemens_rawdat(
     Raises
     ------
     ImportError
-        If the mapVBVD module is not available.
+        If the twixtools module is not available.
 
     Notes
     -----
-    This function requires the mapVBVD module to be installed.
+    This function requires the twixtools module to be installed.
     You can install it using the following command:
-        `pip install pymapVBVD`
+        `pip install gt-twixtools`
     """
     try:
-        from mapvbvd import mapVBVD
+        from twixtools import map_twix, read_twix
     except ImportError as err:
         raise ImportError(
-            "The mapVBVD module is not available. Please install it using "
-            "the following command: pip install pymapVBVD"
+            "The twixtools module is not available. Please install it using "
+            "the following command: pip install gt-twixtools"
         ) from err
-    twixObj = mapVBVD(filename)
+    twixObj = map_twix(read_twix(filename))
     if isinstance(twixObj, list):
         twixObj = twixObj[-1]
-    twixObj.image.flagRemoveOS = removeOS
+    twixObj['image'].flags['remove_os'] = removeOS
     hdr = {
-        "n_coils": int(twixObj.image.NCha),
-        "n_shots": int(twixObj.image.NLin),
-        "n_contrasts": int(twixObj.image.NSet),
-        "n_adc_samples": int(twixObj.image.NCol),
-        "n_slices": int(twixObj.image.NSli),
+        "n_coils": int(twixObj['image'].shape[-2]),
+        "n_shots": int(twixObj['image'].shape[-3]),
+        "n_contrasts": int(twixObj['image'].shape[-4]),
+        "n_adc_samples": int(twixObj['image'].shape[-1]),
+        "n_slices": int(int(twixObj['image'].shape[-5])),
```

Comment:
```suggestion
k: int(v) for k,v in zip(["n_adc_samples", "n_coils", "n_shots", "n_contrasts", "n_slices"], twixObj["image"].shape[::-1]) 
```

Reply:
I wanted to stick with the previous code "format" but sure

Reply:
well at least put them in the correct order :D

Reply:
You're right maybe my goal of "sticking with the previous code and only change func calls etc." was at the cost of "less cleaner code". Let's go your way